taxi/models.py

```diff
@@ -0,0 +1,41 @@
+from django.db import models
+
+from django.contrib.auth.models import AbstractUser
+
+from django.conf import settings
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+    def __str__(self):
+        return f"{self.name} ({self.country})"
+
+    class Meta:
+        ordering = ("name",)
+
+
+class Car(models.Model):
+    manufacturer = models.ForeignKey(
+        Manufacturer, on_delete=models.CASCADE)
+    drivers = models.ManyToManyField(settings.AUTH_USER_MODEL,
+                                     related_name="cars"
```

Comment:
```suggestion
                                     related_name="cars")
```
move a bracket to the top